nitransforms/base.py

```diff
@@ -294,6 +317,127 @@ def _to_hdf5(self, x5_root):
         raise NotImplementedError
 
 
+class TransformChain(TransformBase):
+    """Implements the concatenation of transforms."""
+
+    __slots__ = ['_transforms']
+
+    def __init__(self, transforms=None):
+        """Initialize a chain of transforms."""
+        self._transforms = None
+        if transforms is not None:
+            self.transforms = transforms
+
+    def __add__(self, b):
+        """
+        Compose this and other transforms.
+
+        Example
+        -------
+        >>> T1 = TransformBase()
+        >>> added = T1 + TransformBase() + TransformBase()
+        >>> isinstance(added, TransformChain)
+        True
+
+        >>> len(added.transforms)
+        3
+
+        """
+        self.append(b)
+        return self
+
+    def __getitem__(self, i):
+        """
+        Enable indexed access of transform chains.
+
+        Example
+        -------
+        >>> T1 = TransformBase()
+        >>> chain = T1 + TransformBase()
+        >>> chain[0] == T1
```

Comment:
to show its the same object
```suggestion
        >>> chain[0] is T1
```